core/arch/opnd.h

```diff
@@ -2472,6 +2472,19 @@ DR_API
 bool
 reg_is_simd(reg_id_t reg);
 
+DR_API
+/**
+ * Assumes that \p reg is a DR_REG_ constant.
+ * Returns true iff it refers to an SSE or AVX* register.
```

Comment:
nit: We don't say AVX* anywhere else. I think just s/AVX*/AVX/ should be clear, since you mention xmm, ymm, or zmm below.

Reply:
Done